### PR TITLE
Add CODEOWNERS file

### PR DESCRIPTION
**Pull request by dthaler**
Create a CODEOWNERS file for reviewing changes to the ebpf.io site

Signed-off-by: Dave Thaler <dthaler@microsoft.com>

### CODE REVIEW

File: CODEOWNERS [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# This is a comment.
+# Each line is a file pattern followed by one or more owners.
+
+# These owners will be the default owners for everything in
+# the repo unless a later match takes precedence, and
+# will be requested for review when someone opens a pull request.
+*       @cilium/ebpf-bsc
```
Comment (lmb) [inline CODEOWNERS:7]: This has moved now?

Comment (dthaler) [reply CODEOWNERS]: Yes, updated.

**Review by lmb — APPROVED** [review]

**Comment by tgraf** [comment]
@ebpf-io/ebpf-bsc Are we good to merge this?

**Comment by joestringer** [comment]
@tgraf I think it makes sense for the BSC to review project proposal submissions, but I don't know if it makes sense to make them the codeowners for the entire repository given that there's another group doing development / CSS tweaks / etc.

**Review by borkmann — APPROVED** [review]
LGTM, pls squash all three commits into single one for merge.

**Pull request merged**